tdms/include/interpolation_methods.h

```diff
@@ -48,13 +48,11 @@ double interp3(double *v);
  * @param j_u Greatest j index into the FDTD grid to evaluate the field at
  * @param k_l Least k index into the FDTD grid to evaluate the field at
  * @param k_u Greatest k index into the FDTD grid to evaluate the field at
- * @param I Number of elements in the i direction of the FDTD grid
- * @param J Number of elements in the j direction of the FDTD grid
- * @param K Number of elements in the k direction of the FDTD grid
+ * @param nI,nJ,nK Number of elements in the i,j,k (respectively) direction of the FDTD grid
  *
  * @throws runtime_error In the event that the limits of field extraction are outside the FDTD grid
  */
-void checkInterpolationPoints(int i_l, int i_u, int j_l, int j_u, int k_l, int k_u, int I, int J, int K);
+void checkInterpolationPoints(int i_l, int i_u, int j_l, int j_u, int k_l, int k_u, int nI, int nJ, int nK);
 
 /**
  * @brief Defines our order of preference for the use of the various schemes.
```

Comment:
Would it be possible to have some additional comments explaining what is meant by "interp position = 0", etc., below. I think I understand what is meant, but it would be good to make this explicit. Upon further reading it is clear, however, I think a small additional comment somewhere would make this clearer.

Reply:
No problem - I've added a comment at the end of this comment block, and have included an ASCII diagram similar to those for the cubic interpolation functions.